### PR TITLE
Add resource limitation (CPU + mem) for Deb packaging

### PR DESCRIPTION
**Pull request by arielshaqed**
Tested on Ubuntu Trusty. Not sure memory limitation will work on all current Debian builds.

Fixes issue #3 

@yosih please review & merge.

### CODE REVIEW

File: contrib/netperf/init.d-collectd-td-deb [hunk 1]
```diff
@@ -21,19 +21,41 @@ logfile="/var/log/collectd-td.log"
...
+# when Debian and/or Ubuntu do provide a replacement.
+configure_cgroup() {
+  local cpu=$cgroup_root/cpu/$cgroup_path
+  local mem=$cgroup_root/memory/$cgroup_path
+  mkdir -p $cpu || log_daemon_msg "Failed to create cgroup $cpu"
+  # TODO(arielshaqed): If Debian doesn't have cgroup memory controller, only
+  # do this when $cgroup_root/memory is loaded.
+  mkdir -p $mem || log_daemon_msg "Failed to create cgroup $mem"
+  echo 1000000 > $cpu/cpu.cfs_period_us
+  echo 1500 > $cpu/cpu.cfs_quota_us
```
Comment (yosih) [inline contrib/netperf/init.d-collectd-td-deb]: 10000?

Comment (arielshaqed) [inline contrib/netperf/init.d-collectd-td-deb]: Ouch! That should have been 150000 (we agreed to take up 1.5%).

**Comment by arielshaqed** [comment]
@yosih PTAL

**Comment by yosih** [comment]
LGTM

**Pull request merged**